it/content/src/main/content/jcr_root/content/forms/af/core-components-it/samples/ruleeditor/submit/submitdefaulterrorhandler/.content.xml

```diff
@@ -3,20 +3,28 @@
     jcr:primaryType="cq:Page">
     <jcr:content
         cq:deviceGroups="[/etc/mobile/groups/responsive]"
-        cq:lastModified="{Date}2024-04-10T06:59:23.294Z"
+        cq:lastModified="{Date}2024-06-25T16:30:36.396Z"
         cq:lastModifiedBy="admin"
         cq:template="/conf/core-components-examples/settings/wcm/templates/af-blank-v2"
         jcr:language="en"
         jcr:primaryType="cq:PageContent"
         jcr:title="submitDefaultErrorHandler"
         sling:configRef="/conf/forms/core-components-it/samples/ruleeditor/submit/submitdefaulterrorhandler/"
-        sling:resourceType="forms-components-examples/components/page">
+        sling:resourceType="forms-components-examples/components/page"
+        ignoreTranslationInvalidation="{Boolean}true">
         <guideContainer
             fd:version="2.1"
+            jcr:lastModified="{Date}2024-06-25T16:30:36.392Z"
+            jcr:lastModifiedBy="admin"
             jcr:primaryType="nt:unstructured"
             sling:resourceType="forms-components-examples/components/form/container"
+            actionType="fd/af/components/guidesubmittype/restendpoint"
             dorType="none"
             fieldType="form"
+            schemaType="none"
+            specVersion="0.13.0"
+            textIsRich="true"
+            thankYouMessage="Thank you for submitting the form."
```

Comment:
Why are existing test cases failing ? is there a backward incompatible change

Reply:
Earlier if we leave redirection url on submission empty then submission would fail. Now we get Thank you message, if redirection url is empty.

Reply:
So I changed it to submit to rest endpoint without mentioning the endpoint.

Reply:
Who made this change ? I mean why should submission fail, if something is not configured, a default fallback should exist